pyproject.toml

```diff
@@ -59,6 +59,7 @@ behave = { version = "^1.2.6", optional = true }
 mock = { version = "^4.0.3", optional = true }
 pytest-mock = { version = "^3.7.0", optional = true }
 matplotlib = { version = "^3.5.2", optional = true }
+isolate = "^0.3.0"
```

Comment:
If fal still has its internal package management, it should not depend on isolate if it doesn't really use it, right?